shared/desktop/app/main-window.desktop.js

```diff
@@ -65,7 +65,7 @@ export default function() {
   // app.getLoginItemSettings().restoreState is Mac only, so consider it always on in Windows
   const isRestore =
     !!process.env['KEYBASE_RESTORE_UI'] || app.getLoginItemSettings().restoreState || isWindows
-  const hideWindowOnStart = process.env['KEYBASE_START_UI'] === 'hideWindow'
+  const hideWindowOnStart = process.env['KEYBASE_AUTOSTART'] === '1'
```

Comment:
Did you mean to remove `KEYBASE_START_UI`? It's probably not really useful outside autostart, but thought I'd bring it up.

Reply:
Yeah, I realize this kind of complects all the autostart features together but I wanted to allow people to write their own init and startup scripts, and I thought KEYBASE_AUTOSTART=1 was better than them having to set KEYBASE_START_UI=hideWindow.

Reply:
Also so we can use that same variable for other things if the GUI needs to.